CONTRIBUTING.md

```diff
@@ -93,18 +89,9 @@ GithubService which has everything set up:
 
 #### Requirements
 
-- [podman](https://github.com/containers/libpod)
-- docker
+- ansible
 
 #### Targets
-
-Here are some important and useful targets of [Makefile](/Makefile):
-
-Build a container image for packit service:
-```
-make build
```

Comment:
I don't understand, why the `make build` was removed. We do not plan to run `packit` as a tool in a container?

Reply:
I moved it to packit-service repo since the image is based on httpd and only used as a service.

Do you want it to run in a container?